### PR TITLE
Update generic-polling.md

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
Ready

## Description
Created a troubleshooting section for GenericPolling playbook

### CODE REVIEW

File: docs/playbooks/generic-polling.md [hunk 1]
```diff
@@ -57,3 +57,8 @@ The polling is done only once the result is `finished`. The dt filter will retur
 * Does not run from **playground**.
 * Polling command must support list argument.
 ![image](../doc_imgs/playbooks/66293071-7d168880-e8ee-11e9-9d55-e8ae1e09fe0e.png)
+
+## Troubleshooting
+* **Playbook is "stuck" on `Waiting for polling to complete`:** Since generic polling schedule tasks outside the context of the playbook (not visible in the playbook run), a lot of errors appear only in the War Room. Go to the incident War Room and check for errors or warnings related to GenericPolling tasks.
+* **GenericPolling task is completed but status is still not "finished":** If the timeout was reached, the playbook will succesfully finish even if there are still items that are not completed. Try increasing the timeout value for the GenericPolling task.
```
Comment (Itay4) [inline docs/playbooks/generic-polling.md:63]: i think we can remove the note that says exactly that?

Comment (ShahafBenYakir) [reply docs/playbooks/generic-polling.md:63]: Hmmm maybe remove it?

Comment (ShahafBenYakir) [reply docs/playbooks/generic-polling.md:63]: On a second look, I think it is ok to leave it as id

**Comment by fvigo** [comment]
@ShahafBenYakir there are other things worth mentioning:
- GenericPolling doesn't work properly in the Playground, so testing should be done within an incident
- Sometimes there are race conditions in products where the ID of the first command is not immediately available for polling

**Comment by ShahafBenYakir** [comment]
@fvigo 
Thanks for the info I will add this.

File: docs/playbooks/generic-polling.md [hunk 2]
```diff
@@ -54,6 +54,13 @@ The polling is done only once the result is `finished`. The dt filter will retur
...
+* **The integration returnes an ID not found error when running from GenericPolling, however when running manuualy, it finishes successfully:** Some products cannot handle consequtive requests for querying an action status right after the request to perform the action itself. After you initiate the action, try adding a `sleep` task before calling the **GenericPolling** sub-playbook.
```
Comment (Itay4) [inline docs/playbooks/generic-polling.md:65]:
```suggestion
* **The integration returnes an ID not found error when running from GenericPolling, however when running manualy, it finishes successfully:** Some products cannot handle consecutive requests for querying an action status right after the request to perform the action itself. After you initiate the action, try adding a `Sleep` task before calling the **GenericPolling** sub-playbook.
```

**Review by Itay4 — APPROVED** [review]

**Pull request merged**